Gemfile

```diff
@@ -26,6 +26,7 @@ gem 'font-awesome-sass', '~> 6.1'
 gem 'kaminari' # For pagination
 gem 'sassc-rails' # Use Sass to process CSS
```

Comment:
Good spot @Huiclaire!  Apart from that all looks good to me, seems to work fine on my machine too.
@awalsh3 would you like to remove this gem before we merge, just in case? Or I can add a commit and do that! :)

Reply:
All done, thank you!